app/static/src/scss/includes/_browse-all.scss

```diff
@@ -197,43 +166,29 @@
 
 .govuk-form-group {
   &--browse-all-filter {
-
     margin-bottom: 0;
   }
 
   &__browse-all-form-group {
-
     display:inline-block;
     margin-bottom: 0;
 
   }
-
-
 }
 
+.govuk-link {
+  &--browse-all-filter {
+    margin-left: 1rem;
+    margin-bottom: 0.6rem;
+  }
+}
 
 .govuk-button {
   &__browse-all-filters-form-apply-button {
-
     margin-left: 0;
     margin-bottom: 10px;
     margin-top: 0;
   }
-
```

Comment:
was `browse-all-filters-form-clear-button` unused?

I see we are now applying `govuk-link`, whats the visual difference this causes? They look the same in the screenshots

Reply:
Anchor tag <a> applied for clear filter instead of button